apps/prairielearn/elements/pl-drawing/mechanicsObjects.js

```diff
@@ -3261,9 +3261,9 @@ mechanicsObjects.byType['pl-distributed-load'] = class extends PLDrawingBaseElem
       var initObjTop = obj.top;
 
       var modify = function (subObj) {
-        (subObj.left = initSubObjLeft + obj.left - initObjLeft),
-          (subObj.top = initSubObjTop + obj.top - initObjTop),
-          (subObj.range = obj.range);
+        subObj.left = initSubObjLeft + obj.left - initObjLeft;
+        subObj.top = initSubObjTop + obj.top - initObjTop;
+        subObj.range = obj.range;
```

Comment:
Sanity check: I'm not missing some super obscure reason that this had to be written the way it was before, right?